AutomatedTesting/Gem/PythonTests/Atom/tests/levelLoad_Atom_macbeth_shaderballs.py

```diff
@@ -0,0 +1,79 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+class Tests:
+    macbeth_shaderballs_level_load = (
+        "macbeth_shaderballs level loaded",
+        "P0: macbeth_shaderballs level failed to load")
+    macbeth_shaderballs_enter_game_mode = (
+        "macbeth_shaderballs entered gameplay successfully",
+        "P0: macbeth_shaderballs failed to enter gameplay")
+    macbeth_shaderballs_exit_game_mode = (
+        "macbeth_shaderballs exited gameplay successfully",
+        "P0: macbeth_shaderballs failed to exit gameplay")
+    empty_level_load = (
+        "Empty level loaded successfully, Editor remains stable",
+        "P0: Empty level fails to load, editor is either hung or crashed")
+
```

Comment:
NIT: There should be 2 blank lines between the class and the function, this file has 3.

Reply:
fixed!